modules/@apostrophecms/piece-type/ui/apos/components/AposImageRelationshipEditor.vue

```diff
@@ -90,24 +101,155 @@ export default {
       modalTitle: {
         key: 'apostrophe:editImageRelationshipTitle',
         title: this.title
-      }
+      },
+      groups: [],
+      currentTab: null,
+      tabs: [],
+      fields: {},
+      visibleSchema: [],
+      alignedFields: [ 'width', 'height' ]
     };
   },
   async mounted() {
     this.modal.active = true;
     this.docReady = true;
+
+    this.setVisibleSchema();
+    this.setNestedSchema();
+
+    // this.setGroups();
```

Comment:
Started implementing groups but realized we don't need it for focal points. but seeing the designs we will in the future so I'm gonna create a ticket.

Reply:
We are not using AposSchema here so I am not sure why we would need it unless perhaps you mean a separate implementation of groups for the specific need here.

Reply:
should we remove the comments then?

Reply:
Yes I thought of a better implementation since it's used in different components. That was as a reference but I agree we could remove these comments.